www/src/pages/foundations/responsive.jsx

```diff
@@ -35,7 +35,7 @@ const Responsive = () => {
         {/* eslint-disable-next-line react/jsx-pascal-case */}
         <SEO title="Responsive" />
         <h1>Responsive</h1>
-        <h3>Available breakpoints</h3>
+        <h2>Available breakpoints</h2>
```

Comment:
Can we also make the "Basic Usage" heading below the same heading level as this one, an `h2`?

Reply:
Done